node/collation-generation/src/lib.rs

```diff
@@ -139,16 +139,17 @@ impl CollationGenerationSubsystem {
 				msg: CollationGenerationMessage::Initialize(config),
 			}) => {
 				if self.config.is_some() {
-					log::error!(target: LOG_TARGET, "double initialization");
+					tracing::error!(target: LOG_TARGET, "double initialization");
 				} else {
 					self.config = Some(Arc::new(config));
 				}
 				false
 			}
 			Ok(Signal(BlockFinalized(_))) => false,
 			Err(err) => {
-				log::error!(
+				tracing::error!(
```

Comment:
Somewhere the logging has this prefix, somewhere not; probably we should remove it everywhere and just use `warn!` etc
```suggestion
				error!(
```

Reply:
I prefer retaining the scope prefix: it means you can just add a `tracing::warn!` entry, for example, without having to double-check whether or not there are any other `warn` instances in the file and update the `use` block in the headers.

Reply:
8ff0fe7